### DISCUSSION TITLE
[doris-2.0.2] be莫名挂掉后无法启动，be.out日志循环打印"Exception: java.lang.StackOverflowError thrown from the UncaughtExceptionHandler in thread "process reaper""

### DISCUSSION THREAD

**Discussion by JoshuaNocturne** [apache/doris#29212] [A - General / Q&A] [answered]
三个be节点同时挂掉，be日志中未见任何异常，重新启动be日志循环打印：
Exception: java.lang.StackOverflowError thrown from the UncaughtExceptionHandler in thread "process reaper"
Exception in thread "process reaper" java.lang.StackOverflowError

无法正常启动，dmesg未见oom

**Comment by LemonLiTree** [comment, marked as answer]
java home 配置java 8目录了吗

**Reply by JoshuaNocturne** [reply]
jdk之前一直使用的是11也没见什么问题，必须换8吗？

**Reply by JoshuaNocturne** [reply]
改用jdk8后正常了，谢谢，不过还是想多问一句为什么jdk11会启动报错呢

**Reply by LemonLiTree** [reply]
兼容这块可能还有问题。。

**Reply by ihadoop** [reply]
有详细的堆栈信息么
